sphinxgallery/__init__.py

```diff
@@ -1,8 +1,8 @@
 """Sphinx Gallery
 """
 import os
-__version__ = '0.0.7'
+__version__ = '0.0.9'
```

Comment:
I am not sure about the origin of this diff but I can see that sphinx-gallery 0.0.8 has already been released on pypi. 

I am going to hazard a wild guess and assume that you released from a local modification which should be avoided. Tagging the release point with git is also a good idea for the future releases. I can see that you did that for a few releases in the past.